XpringKit/Utils.swift

```diff
@@ -1,66 +1,132 @@
-  /// - Returns: A new x-address if inputs were valid, otherwise undefined.
-  public static func encode(classicAddress: Address, tag: UInt32? = nil, isTest: Bool = false) -> Address? {
-    return javaScriptUtils.encode(classicAddress: classicAddress, tag: tag, isTest: isTest)
-  }
+    /// Encode the given classic address and tag into an x-address.
+    ///
+    /// - SeeAlso: https://xrpaddress.info/
+    ///
+    /// - Parameters:
+    ///   -  classicAddress: A classic address to encode.
+    ///   - tag: An optional tag to encode. Defaults to nil.
+    ///   - isTest Whether the address is for use on a test network, defaults to `false`.
+    /// - Returns: A new x-address if inputs were valid, otherwise undefined.
+    public static func encode(classicAddress: Address, tag: UInt32? = nil, isTest: Bool = false) -> Address? {
+        return javaScriptUtils.encode(classicAddress: classicAddress, tag: tag, isTest: isTest)
+    }
 
-  /// Decode a classic address from a given x-address.
-  ///
-  /// - SeeAlso: https://xrpaddress.info/
-  ///
-  /// - Parameter xAddress: The xAddress to decode.
-  /// - Returns: a tuple containing the decoded address,  tag and bool indicating if the address was on a test network.
-  public static func decode(xAddress: Address) -> (classicAddress: String, tag: UInt32?, isTest: Bool)? {
-    return javaScriptUtils.decode(xAddress: xAddress)
-  }
+    /// Decode a classic address from a given x-address.
+    ///
+    /// - SeeAlso: https://xrpaddress.info/
+    ///
+    /// - Parameter xAddress: The xAddress to decode.
+    /// - Returns: a tuple containing the decoded address,  tag and bool indicating if the address was on a test network.
+    public static func decode(xAddress: Address) -> (classicAddress: String, tag: UInt32?, isTest: Bool)? {
+        return javaScriptUtils.decode(xAddress: xAddress)
+    }
 
-  /// Convert the given transaction blob to a transaction hash.
-  ///
-  /// - Parameter transactionBlobHex: A hexadecimal encoded transaction blob.
-  /// - Returns: A hex encoded hash if the input was valid, otherwise undefined.
-  public static func toTransactionHash(transactionBlobHex: Hex) -> Hex? {
-    return javaScriptUtils.toTransactionHash(transactionBlobHex: transactionBlobHex)
-  }
+    /// Convert the given transaction blob to a transaction hash.
+    ///
+    /// - Parameter transactionBlobHex: A hexadecimal encoded transaction blob.
+    /// - Returns: A hex encoded hash if the input was valid, otherwise undefined.
+    public static func toTransactionHash(transactionBlobHex: Hex) -> Hex? {
+        return javaScriptUtils.toTransactionHash(transactionBlobHex: transactionBlobHex)
+    }
+
+    /// Convert the given string to a bytes array UInt8.
+    ///
+    /// - Parameter hex: A string.
+    /// - Returns: A byte array in [UInt8].
+    public static func toByteArray(hex: String ) -> [UInt8] {
+        // remove "-" from Hexadecimal
+        let hexString = hex.removeWord( "-" )
```

Comment:
Remove spaces after `(` and before `)`. Here and elsewhere in this method. This is consistent with style throughout the existing code.